lib/integration/queryManager.ts

```diff
@@ -0,0 +1,30 @@
+import { logMessage } from "../logger";
+import dbConnector from "./dbConnector";
+import { QueryArrayResult } from "pg";
+
+const queryManager = {
+  async executeQuery(sql: string, params: [string]): Promise<QueryArrayResult<unknown[]>> {
+    try {
+      // Get a client
+      const client = dbConnector();
+      //Stablish a connexion
+      await client.connect();
+      if (!sql) throw new Error("Invalid query param");
+
+      return await client.query(sql, params);
+    } catch (error: unknown) {
+      logMessage.error(`{"error: "${error}"}`);
+      throw new Error(error as string);
+    }
+  },
+
+  getResult(data: QueryArrayResult): unknown[][] {
+    if (data.rowCount > 0) {
+      return data.rows.map((record) => {
+        return record;
```

Comment:
What does this function do exactly ? The map here just returns the same array !

Reply:
It's been already refactored. 
The concern is to only extract the result from metadata  that is returned.

`rowCount: 1
oid: null
rows: [object Object]
fields: [object Object]
_parsers: function noParse (val) {
  return String(val);
}
_types: [object Object]
RowCtor: null
rowAsArray: false`